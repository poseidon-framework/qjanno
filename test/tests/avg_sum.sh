qhs -H -O "SELECT avg(bar),sum(bar) FROM basic.csv"
