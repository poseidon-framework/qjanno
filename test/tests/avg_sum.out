avg(bar) sum(bar)
2.0 4
